### PR TITLE
Added Link provider

### PR DESCRIPTION
**Pull request by Doc415**
gettink link from appsettings.json

### CODE REVIEW

File: TCSA.V2/Program.cs [hunk 1]
```diff
@@ -61,6 +62,7 @@
 
     })
     .AddIdentityCookies();
+LinkProvider.DiscordLink=builder.Configuration.GetConnectionString("DiscordLink") ?? throw new InvalidOperationException("Discord link not found.");
```
Comment (stevietv) [inline TCSA.V2/Program.cs:65]: should this really be stored in a connectionstring part of the appsettings, or it its own section.
you could do it by adding a section `LinkProvider` to the appsettings.json with a key for `DiscordLink`

File: TCSA.V2/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,38 +1,39 @@
 {
```
Comment (stevietv) [inline TCSA.V2/Properties/launchSettings.json:1]: this file shouldn't be included in the PR

**Review by stevietv — CHANGES_REQUESTED** [review]
a few comments of input from me :)

File: TCSA.V2/Properties/launchSettings.json (hunk 2 above)
Comment (Doc415) [reply TCSA.V2/Properties/launchSettings.json]: Changes completed

**Comment by Doc415** [comment]
> a few comments of input from me :)

Thanks 🙋‍♂️

**Review by stevietv — APPROVED** [review]
